internal/pkg/aws/sts/sts.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package sts wraps AWS Security Token Service (STS) API functionality.
+package sts
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/aws/aws-sdk-go/service/sts/stsiface"
+)
+
+// Client wraps the internal sts client.
+type Client struct {
```

Comment:
what's the main reason we need to wrap STS? Is the AWS SDK not high-level enough?

Reply:
Mostly dependency isolation.